pom.xml

```diff
@@ -126,8 +126,8 @@
 		<spotbugs.version>4.7.3</spotbugs.version>
 		<maven.spotbugs.version>4.7.3.0</maven.spotbugs.version>
 		<strimzi-oauth.version>0.14.0</strimzi-oauth.version>
-		<opentelemetry.alpha-version>1.19.0-alpha</opentelemetry.alpha-version>
-		<opentelemetry.version>1.19.0</opentelemetry.version>
+		<opentelemetry.version>1.34.1</opentelemetry.version>
+		<opentelemetry.instrumentation.version>1.32.0-alpha</opentelemetry.instrumentation.version>
```

Comment:
why using two different versions and not aligning them?

Reply:
AFICT there is nothing to be gained from aligning them (happy to be proven wrong) so went with latest and greatest for each.

Reply:
True, but it had backwards incompatible changes I didn't fancy reasoning about. I will align them when I remove the bom's

Reply:
About using 2.0 we have to pay more attention.
It seems there are a lot of breaking changes https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v2.0.0
The main ones impacting us would be:

* defaulting the OTLP protocol from `grpc` to `http/protobuf`. To keep backward compatibility on our side, it means we should "force" grpc in the bridge `otel.exporter.otlp.protocol=grpc` if the user doesn't set `OTEL_EXPORTER_OTLP_PROTOCOL` env var explicitely.
* we should explicitely disable the logs exporter `otel.logs.exporter=none` as we do for the metrics exporter already.

Reply:
I can bump to 2.0 if you would prefer.

Reply:
Do you want to move to 2.0 now or not? I've addressed the logs exporter point as that applied without moving to 2.0.0.

Reply:
I would avoid moving to a 2.x version for now. It would be anyway only for the instrumented Kafka client but I would leave using a 1.3x version for now.